functions.php

```diff
@@ -284,7 +284,7 @@ public function people_sort( $people ) {
 
 		if ( 1 === ( count( $people ) % 2 ) ) {
 			$person = new stdClass();
-			$person->title = '';
+			$person->title = (object) array( 'rendered' => '' );
```

Comment:
In hindsight, we could probably handle this all as an array and then do `$people[] = (object) $person`, but I think this is a good bandaid.